apps/src/templates/studioHomepages/JoinSectionArea.jsx

```diff
@@ -58,9 +58,7 @@ export default function JoinSectionArea({
         sectionCapacity={sectionResults.sectionCapacity}
       />
       <JoinSection
-        enrolledInASection={
-          joinedStudentSections?.length > 0 || joinedPlSections?.length > 0
-        }
+        enrolledInASection={true}
```

Comment:
Yeah, it is.
In designs we have this section with solid border instead of dashed. However I decided to keep the possibility to make `apps/src/templates/studioHomepages/JoinSection.jsx` dashed not to break anything elsewhere

Reply:
ahhh, i just misunderstood what the `enrolledInASection` prop was doing (it seems inappropriately named). makes sense!